src/func.d

```diff
@@ -2169,13 +2169,12 @@ public:
             sc2.callSuper = 0;
             sc2.pop();
         }
-        if (needsClosure())
+
+        if (checkClosure())
         {
-            if (setGC())
-                error("@nogc function allocates a closure with the GC");
-            else
-                printGCUsage(loc, "using closure causes GC allocation");
+            //errors = true;
```

Comment:
Currently all error occurrences in `FuncDeclaration.semantic3` is reflected into `FuncDeclaration.errors` variable by using `global.errors` at the end of `semantic3`. But, Walter is continuously saying that the it's a bad design and should be removed in the future. I think too.
Therefore, although today the line `errors = true;` is just a dead code, but it would be enabled when we'll finally remove `global.errors`.